doc/smart-answer-flow-development/creating-a-new-smart-answer.md

```diff
@@ -1,8 +1,38 @@
 # Creating a new Smart Answer
 
-This walks through the basics of creating a new Smart Answer.
+There are two methods for creating a new smart answer:
+  1. automatically - via a rails generator, and
+  2. manually.
 
-## 1. Create a new skeleton flow
+## Using the rails `smart_answer` generator
+
+To automatically generate a skeleton smart answer use the provided rails
+generator...
+
+```bash
+$ rails generate smart_answer example_smart_answer
+```
+
+This will create an example flow, calculator, questions, outcome and landing
+pages.
+
+The generated files do not constitute a complete set of all possible question
+types, and configuration options. However, they should be enough to get you
+started and to show how the various files work together. 
+
+To see a list of the files that the generator will create use either the
+`--help` or `--pretend` options, like this...
+
+```bash
+$ rails generate smart_answer --help
+$ rails generate smart_answer my_smart_answer -p
+```
```

Comment:
Really picky, but it might no be obvious that `-p` is the same as `--pretend`. So if you use `--pretend` in the description (lines 23-24), then `--pretend` should be used here too.

Reply:
Thanks @reggieb - good spot.  I did see it earlier and forgot to change it 😁